lifecycle/src/lifecycle_service_client_py.py

```diff
@@ -75,24 +79,24 @@ def get_available_transitions(lifecycle_node):
     req = GetAvailableTransitions.Request()
     cli.call(req)
     cli.wait_for_future()
-    print('%s has %u available transitions'
-          % (lifecycle_node, len(cli.response.available_transitions)))
+    node.get_logger().info(
+        '%s has %u available transitions'
+        % (lifecycle_node, len(cli.response.available_transitions)))
     for transition in cli.response.available_transitions:
-        print('Transition id: %u\tlabel: %s'
-              % (transition.transition.id, transition.transition.label))
-        print('\tStart id: %u\tlabel: %s'
-              % (transition.start_state.id, transition.start_state.label))
-        print('\tGoal  id: %u\tlabel: %s'
-              % (transition.goal_state.id, transition.goal_state.label))
+        node.get_logger().info(
+            'Transition id: %u\tlabel: %s'
+            % (transition.transition.id, transition.transition.label))
+        node.get_logger().info(
+            '\tStart id: %u\tlabel: %s'
+            % (transition.start_state.id, transition.start_state.label))
+        node.get_logger().info(
+            '\tGoal  id: %u\tlabel: %s'
+            % (transition.goal_state.id, transition.goal_state.label))
 
 
 def main(service_type, lifecycle_node, change_state_args='', args=None):
     rclpy.init(args=args)
 
-    if not rclpy.ok():
```

Comment:
This is not necessary anymore because init would raise on failure ?

Reply:
yep, thanks for making it explicit :smile: